src/Components/Rides/RiderSignup/RiderSignup.tsx

```diff
@@ -4,13 +4,17 @@
 
 import React, { useState } from 'react';
 import { Button, Col, Container, Form, Row } from 'react-bootstrap';
+import ReactModal from 'react-modal';
+import { IconContext } from 'react-icons/lib';
+import { BiHelpCircle } from 'react-icons/bi';
 import { Turnarounds } from '../../../Constants/RidesConstants';
-
+import pickupLocations from '../../../images/pickupLocations.png';

```

Comment:
nit: I think it would be better to use absolute paths in general. The relative path above probably didn't set a good example though - would be great if you could change that too to be absolute. :^)